NitroxServer-Subnautica/Communication/IpcHost.cs

```diff
@@ -57,13 +57,15 @@ public async Task<string> ReadStringAsync(CancellationToken cancellationToken =
         await server.ReadExactlyAsync(sizeBytes, cancellationToken);
         byte[] stringBytes = new byte[BitConverter.ToUInt32(sizeBytes)];
         await server.ReadExactlyAsync(stringBytes, cancellationToken);
+
         return Encoding.UTF8.GetString(stringBytes);
     }
 
     public void Dispose()
     {
         commandReadCancellation?.Cancel();
         server.Dispose();
+        GC.SuppressFinalize(this);
```

Comment:
This type doesn't have a finalizer so afaik it does nothing.
https://stackoverflow.com/a/19078623/1277156

Reply:
I've removed this line